src/Tasks/Microsoft.NET.Build.Extensions.Tasks/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.ConflictResolution.targets

```diff
@@ -25,7 +25,8 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target Name="_HandlePackageFileConflicts"
           BeforeTargets="$(_HandlePackageFileConflictsBefore)"
           AfterTargets="$(_HandlePackageFileConflictsAfter)"
-          DependsOnTargets="GetFrameworkPaths;GetReferenceAssemblyPaths">
+          DependsOnTargets="GetFrameworkPaths;GetReferenceAssemblyPaths"
+          Condition="'@(Reference)' != '' Or '@(ReferencePath)' != ''">
```

Comment:
It feels wrong that this conditions on ReferencePath but doesn't use ReferencePath.